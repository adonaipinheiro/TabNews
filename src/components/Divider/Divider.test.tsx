import {render} from '@testing-library/react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import {Divider} from './';

describe('<Divider />', () => {
  it('should renders correctly with size and horizontal props', () => {
    const divider = renderer.create(
      <Divider size={2} horizontal={false} />,
    );

    expect(divider.root.props).toEqual({size: 2, horizontal: false});
  });

  it('should renders correctly with size and vertical props', () => {
    const divider = renderer.create(
      <Divider size={2} horizontal={true} />,
    );

    expect(divider.root.props).toEqual({size: 2, horizontal: true});
  });

  it('should Divider component renders correctly', () => {
    const {toJSON} = render(<Divider size={2} horizontal={false} />);

    expect(toJSON()).toMatchSnapshot();
  });
});
