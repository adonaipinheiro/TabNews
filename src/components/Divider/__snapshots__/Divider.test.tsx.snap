// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Divider /> should Divider component renders correctly 1`] = `
<View
  style={
    {
      "height": 2,
      "width": 0,
    }
  }
/>
`;
